<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>APIWindowClass</class>
 <widget class="QWidget" name="APIWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>101</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Get data from IP or URL</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="APIDescribeLabel">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>10</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Insert only one IP address or URL below and click Ok button</string>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptAPIButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="StatusDescribeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Status: </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
